doc/port-si/port-si-per-speed.md

```diff
@@ -0,0 +1,267 @@
+
+![Alt text](<json template-1.png>)
+
+
+## How are we going to use this json?
+
+The flow of using of this json will be referred to as the **_Notify-Media-setting-Process_**:
+
+![Notify Media Setting Process](Notify-Media-Setting-Process.drawio.svg)
+
+
+During the **_Notify-Media-setting-Process_** two things occur:
+
+1. STATE_DB is updated with sfp, dom and pm data.
+2. The APP_DB is updated with the connected module SI parameters. This is achieved through the notify_media_settings() function, which uses the pre-parsed media_settings.json file to write its contents to APP_DB: First, a key is composed for performing the JSON lookup, based on the media type, specification compliance, length, and lane speed. Then, the lookup is performed using that key, and the relevant data is extracted from the JSON and stored in APP_DB.
+
+    Before transitioning to independent mode, SERDES SI parameters were never applied because the media_settings.json file was not found on the platform. In the new mode, media_settings.json will be present on every platform, but it will only be utilized when it contains an entry for a plugged-in CMIS module (QSFP-DD, OSFP). If it does not have a suitable entry for the connected module, nothing will be written to APP_DB to maintain backward compatibility with the legacy flow.
+
+    In order for the FW to configure the ASIC in the case of non-CMIS modules, instead of SONiC, it is crucial to ensure that the           media_settings.json file satisfies the following two requirements:
+    - The media_settings.json file should only contain data related to CMIS modules. Signal Integrity (SI) parameters for non-CMIS modules should not be present in this JSON file.
+
+    - The media_settings.json file must not include a 'default' section. Normally, the lookup process in the media_settings.json file allows for a default section. SI parameters, defined in this section, will be written to the APP_DB if there is no matching entry found for any other module in the JSON file. However, since we do not intend SONiC configuration to support non-CMIS modules, it is necessary for the media_settings.json file to exclude the default section.
+
+    </br>Ports OrchAgent listens to changes in APP_DB, so when the APP_DB is updated with SI parameters, PortsOrchagent is triggered. Based on the data found in APP_DB, PortsOA creates a vector that contains the SI values for a certain port and passes it as a whole to the SAI. Eventually, they will write it to the SLTP register in PHY upon receiving ADMIN_UP.  
+  </br></br></br>
+
+
+## Port SI configuration (flows)
+
+![Port SI Flows](<Port SI Flows.drawio.svg>)  
+*** Risk: Currently the loading of media_settings.json is skipped in case fast-reboot is enabled. There's an open issue that is under discussion with Microsoft to revert this commit: https://github.com/sonic-net/sonic-platform-daemons/pull/221. 
```

Comment:
Please capture this under Warmboot and Fastboot support sections.

Reply:
done